packages/slider/src/RangeSlider.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Range from './rc-slider/Range'
+import { space, color } from 'styled-system'
+import styleSlider from './styleSlider'
+
+const RangeSlider = styleSlider(Range)
+
+RangeSlider.defaultProps = {
+  allowCross: false,
+  color: 'blue'
+}
+
+RangeSlider.propTypes = {
+  ...space.propTypes,
+  ...color.propTypes
```

Comment:
I don't think we should be using `color` from styled-system anymore

Reply:
This version of pcln-slider does not require DS 2.3.1 yet, so it should be ok for now right?